utils/tangleid/CoreAPI.js

```diff
@@ -1,3 +1,9 @@
+function handleErrors(response) {
```

Comment:
Is it generic enough? If so, shall we regard it as utility function and move to toplevel?

Reply:
No. It's only used for `fetch` function.
Just keep it here would be nice.

Reply:
@ender503 Can/Should we build a more general exception handler ?

Reply:
It looks like a utility function, it would be great if we could move it to another file, ex: utils/handleErrors.js

Reply:
But the `response.ok` usage is limited to `fetch` function. 
Is there any other function use this to imply function work well ?

Reply:
I think I'll keep it here this time.